src/components/funnels/funnels/register/steps/ticket/level.tsx

```diff
@@ -0,0 +1,94 @@
+/** @jsxImportSource @emotion/react */
+
+import styled from '@emotion/styled'
+import { Localized } from '@fluent/react'
+import { RouteComponentProps } from '@reach/router'
+import { Controller } from 'react-hook-form'
+import { DateTime } from 'luxon'
+import { RadioGroup, Select } from '@eurofurence/reg-component-library'
+import { useSiteMetadata } from '~/hooks/queries/site-metadata'
+import TicketLevelCard from './level/card'
+import TicketLevelAddon from './level/addon'
+import FullWidthRegisterFunnelLayout from '~/components/funnels/funnels/register/layout/form/full-width'
+import { ChangeTicketLevel, SubmitTicketLevel } from '~/state/actions/register'
+import { TicketLevel as TicketLevelModel } from '~/state/models/register'
+import { useFunnelForm } from '~/hooks/funnels/form'
+
+const sizes = [
+	{ value: 'S', label: 'S' },
+	{ value: 'M', label: 'M' },
+	{ value: 'L', label: 'L' },
+	{ value: 'XL', label: 'XL' },
+	{ value: 'XXL', label: 'XXL' },
+]
+
+const sizesByValue = new Map(sizes.map(size => [size.value, size]))
```

Comment:
Maybe this should be moved to a constants + utils file? Seems like it's just data and a logic function

Reply:
I could actually probably move the list of t-shirt sizes to the site config.